docs/quickstart.md

```diff
@@ -54,9 +54,11 @@ Once you have Helm ready, you can initialize the local CLI and also
 install Tiller into your Kubernetes cluster in one step:
 
 ```console
-$ helm init
+$ helm init --max-history 200
 ```
 
+**TIP:** Setting `--max-history` on helm init is recommended as configmaps and other objects in helm history can grow very large if not purged by max limit.
+
```

Comment:
While I like this tip I think that _very large_ can be interpreted as large in size individually or great in number. People doing the quickstart don't tend to know how Helm works. Could the language be changed to note that without a max history the history is kept forever which is a large number of records. Or, something to make it more clear

Reply:
Okay, added the language from your comment. Let me know if it is clearer now.